share/functions/abbr.fish

```diff
@@ -85,7 +85,7 @@ function abbr --description "Manage abbreviations"
 		set key $kv[1]
 		set value $kv[2]
 		# ensure the key contains at least one non-space character
-		if not string match -qr "\w" -- $key
+		if not string match -qr "[^\s]" -- $key
```

Comment:
Well, sure, technically you're right. But with this change it's not a regression any more. So it's good to go for 2.3.0.